activerecord/test/cases/enum_test.rb

```diff
@@ -956,4 +956,17 @@ def self.name
   ensure
     ActiveRecord::Base.logger = old_logger
   end
+
+  test "enum can return a string inquirer instead of a string" do
+    klass = Class.new(ActiveRecord::Base) do
+      self.table_name = "books"
+      enum status: [:proposed, :written], _inquirable: true
```

Comment:
Shouldn't the syntax be `inquirable`? Without the underscore "_"?

Reply:
There are two possible syntaxes for enums now. The old one which I used, and the new one that you mentioned, but then the entire code would be:
```
enum :status, [:proposed, :written], inquirable: true
```

The current implementation works with both, but I used the same format that was already in the tests. I can switch, or I can do both. Not sure what would be better.